src/srtctl/cli/submit.py

```
@@ -69,82 +69,35 @@ def render_commands_file(backend, sglang_config_path: Path, output_path: Path) -
-        print(f"\nJob Name: {self.job_name}")
-        print(f"Output Directory: {self.output_dir}")
-        print("\nGenerated Files:")
-        print("  - config.yaml          (resolved config with defaults)")
-        if self.sglang_config_path:
-            print("  - sglang_config.yaml   (SGLang flags)")
-        print("  - commands.sh          (full bash commands)")
-        print("  - metadata.json        (submission info)")
-        print("\nTo see what commands would run:")
-        print(f"  cat {self.output_dir}/commands.sh")
-        print("\n" + "=" * 60 + "\n")
+def run_dry_run(config: dict, backend, sglang_config_path: Path = None) -> Path:
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)